### ISSUE TITLE
Failed to update JavaScriptCore

### ISSUE THREAD

**Issue by hax** [devsnek/esvu#24]
Not sure what happened...

```
➜  ~ esvu update jsc
esvu ❯ version 1.2.3
JavaScriptCore ❯ Checking version...
esvu ✖ TypeError: Cannot read property '1' of null
    at Function.resolveVersion (/Users/hax/.npm-global/lib/node_modules/esvu/src/engines/javascriptcore.js:62:23)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Function.install (/Users/hax/.npm-global/lib/node_modules/esvu/src/installer.js:43:21)
    at async updateEngine (/Users/hax/.npm-global/lib/node_modules/esvu/src/bin.js:127:3)
    at async main (/Users/hax/.npm-global/lib/node_modules/esvu/src/bin.js:162:7)
esvu ✖ Some engines were not installed.
```

**Comment by devsnek** [comment]
@hax i've published version 1.2.4, hopefully that fixes issues you're seeing?

**Comment by hax** [comment]
Thank you. It works. (though I found I can't run jsc on my mac anymore because new version of jsc require new mac os version so I got `____chkstk_darwin: undefined` error 😓 ).
